.gitignore

```diff
@@ -31,6 +31,9 @@ CATKIN_IGNORE*
 .project
 .cproject
 
+# vscode folder
+.vscode
```

Comment:
@joao-pm-santos96 for your information: you can use a global gitignore file so that you don't have to add a new rule in every git project you create.

More info here: https://stackoverflow.com/questions/7335420/global-git-ignore